DFC.Api.Content.UnitTests/Files/JobProfileRecordResponse_2.json

```diff
@@ -0,0 +1,129 @@
+﻿{
+    "Values": {
+      "{ properties:sourceNodeWithOutgoingRelationships.properties, links:sourceNodeWithOutgoingRelationships.links}": {
+        "links": [
+          {
+            "rel": [ "Resource", "ncs__WorkRoute" ],
```

Comment:
Perhaps we should strip out Resource, and use the map to work back to WorkRoute (i.e. so ncs__ disappears).

Reply:
This is just the response from Neo, the actual output to the front-end gets passed through formatter that removes ncs__ prefixes. We can remove the resource label as well and just have one though that makes sense